components/ActionSheet/src/private/MDCActionSheetItemTableViewCell.m

```diff
@@ -148,14 +148,13 @@ - (void)layoutSubviews {
   _titleLeadingConstraint.constant = leadingConstant;
   CGFloat width = CGRectGetWidth(self.contentView.frame) - leadingConstant - TitleTrailingPadding;
   _titleWidthConstraint.constant = width;
-
-  _imageView.image = _itemAction.image;
+  _imageView.image = [_itemAction.image imageWithRenderingMode:UIImageRenderingModeAlwaysTemplate];
 }
 
 - (void)setAction:(MDCActionSheetAction *)action {
   _itemAction = [action copy];
   _textLabel.text = _itemAction.title;
-  _imageView.image = _itemAction.image;
+  _imageView.image = [_itemAction.image imageWithRenderingMode:UIImageRenderingModeAlwaysTemplate];
```

Comment:
I don't believe that will be needed, if it is then I will add it at a later date. This will have to be on the action items and if that is the case then tint color, actionTextColor, actionFont should all be on the action. As for now I believe maintaining this API I have proposed is the best solution.

Reply:
Do you mean you will remove it at a later date? I'm proposing that the UIImage provided be the one directly applied and that we don't create a second image with a new rendering mode. This matches the behavior of UIKit and MDC components behave.

Reply:
If you don't set the `renderingMode` then the image will not be the correct color when you apply `tintColor` I can remove this line but in doing so it doesn't allow clients to set a `tintColor` on the images.

Reply:
Clients can set the rendering mode. This matches what our other components do. The `tintColor` applied will be the one from the UIImageView, either inherited or explicitly applied.